kubectl-fdb/cmd/plugin_version_checker.go

```diff
@@ -0,0 +1,133 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/go-retryablehttp"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+// VersionChecker interface to help us mock test the version checker
+type VersionChecker interface {
+	getLatestPluginVersion() (string, error)
+}
+
+// RealVersionChecker will do actual call to GitHub API to get version
+type RealVersionChecker struct {
+	VersionChecker
+}
```

Comment:
```suggestion
type RealVersionChecker struct {}
```

It's not needed to add the interface to the struct. The Go compiler will detect that this struct implements all methods of the interface, e.g. https://go.dev/tour/methods/10.